cypress/integration/WhereStartLookingPage.feature

```diff
@@ -23,3 +23,26 @@ Feature: Where should we start looking Page
     When I select "one to three countries" radio button
     When I click the "check eligibility" button
     Then I am on the "where should we start looking" page
+
+  Scenario: Selecting the i'm not sure where to look and clicking check eligibility navigates to the eligible one page
+    When I select "i'm not sure they could be anywhere" radio button
+    When I click the "check eligibility" button
+    Then I am on the "eligible for one not sure" page
+
+  Scenario: Selecting the i'm not sure where to look, clicking check eligibility, clicking back goes to where to start looking
+    When I select "i'm not sure they could be anywhere" radio button
+    When I click the "check eligibility" button
+    When I click the back link
+    Then I am on the "where should we start looking" page
+
+  Scenario: Selecting the i'm not sure where to look, clicking check eligibility, Im not sure advice is shown
+    When I select "i'm not sure they could be anywhere" radio button
+    When I click the "check eligibility" button
+    Then I should see the I'm not sure advice
+
+  Scenario: Selecting only inactive countries, clicking check eligibility, no active countries advice is shown
+    When I select "one to three countries" radio button
+    When I select "AF" in the "country-1" select box
```

Comment:
I would prefer to use the text that people see in the select menu instead of the value, but it's not a deal breaker given the MVP timescales.

Reply:
Maybe add as a tech debt to go and fix rather than as part of this PR?

Reply:
Definitely not this PR, maybe if you have time later, probably not in reality ;-)